sysutils/virtualbox/pkg-descr

```diff
@@ -0,0 +1,9 @@
+These additions are for installation inside a FreeBSD guest.
+
+VirtualBox is a family of powerful x86 virtualization products for
+enterprise as well as home use. Not only is VirtualBox an extremely
+feature rich, high performance product for enterprise customers, it
+is also the only professional solution that is freely available as
+Open Source Software under the terms of the GNU General Public License.
+
+: http://www.virtualbox.org
```

Comment:
```suggestion
WWW: https://www.virtualbox.org
```

Reply:
In fact it looked strange to me: but I copied exactly what I found in the same file under sysutils/vmware (which you just fixed it with opnsense/plugins@566a38480d4feb506ee4e4ee5e465be5c2da9e2a), thanks!

Reply:
No big deal, just gotta make it prettier :)